### PR TITLE
Create a new subproject for SpotBugs Ant task

### PR DESCRIPTION
**Pull request by KengoTODA**
This pull request adds a missing subproject to release Ant task for SpotBugs.
I referred following part of `build.xml` to write `build.gradle`.

* https://github.com/findbugsproject/findbugs/blob/d1e60f8/findbugs/build.xml#L363-L373

### CODE REVIEW

File: findbugs/build.gradle [hunk 1]
```diff
@@ -178,7 +178,7 @@ task unzipDist(type:Copy, dependsOn:distZip) {
 task smokeTest {
   dependsOn unzipDist
   doLast {
-    ant.taskdef(name:'spotbugs', classname:'edu.umd.cs.findbugs.anttask.FindBugsTask', classpath:jar.outputs.files.asPath)
+    ant.taskdef(name:'spotbugs', classname:'edu.umd.cs.findbugs.anttask.FindBugsTask', classpath:project(':spotbugs-ant').jar.outputs.files.asPath)
```
Comment (jsotuyod) [inline findbugs/build.gradle]: if you are to do this, you should add `project(':spotbugs-ant')` to this task's `dependsOn` at line 179.

File: findbugs/build.gradle [hunk 2]
```diff
@@ -58,7 +62,10 @@ tasks.withType(Jar).all {
 
 // Manually define what goes into the default jar, since it's not only main sourceset
 jar {
+  dependsOn project(':spotbugs-annotations').compileJava
```
Comment (jsotuyod) [inline findbugs/build.gradle]: this shouldn't be necessary, you are already adding it as a compile dependency

File: findbugs/build.gradle [hunk 3]
```diff
@@ -189,6 +196,7 @@ task smokeTest {
       sourcePath(path:'src/main/java:src/gui:src/test/java:src/tools')
       'class'(location:project.tasks['compileJava'].destinationDir)
       auxClasspath(path:project.fileTree(dir:'lib', include:'**/*.jar').asPath)
+      configurations.compile.each { File file -> auxClasspath(path:file.path) }
```
Comment (jsotuyod) [inline findbugs/build.gradle]: `auxClasspath(configurations.compile.asPath)`?

Also, this should already include the lib dir, so line 198 is not needed anymore.

Comment (KengoTODA) [inline findbugs/build.gradle]: #109 deleted jar file from `findbugs/lib` directory, so we need to add `configurations.compile` explicitly.

Comment (KengoTODA) [inline findbugs/build.gradle]: > auxClasspath(configurations.compile.asPath)?

Will trigger following build failure:

```
* What went wrong:
Execution failed for task ':findbugs:smokeTest'.
> The <path> type doesn't support nested text data ("/Users/k....0.1.jar").
```

Comment (jsotuyod) [inline findbugs/build.gradle]: Weird... non the less, `configurations.compile` includes `project.fileTree(dir:'lib', include:'**/*.jar')`, so you can delete `auxClasspath(path:project.fileTree(dir:'lib', include:'**/*.jar').asPath)`

Comment (KengoTODA) [inline findbugs/build.gradle]: Deleted, thanks for your review!

**Comment by KengoTODA** [comment]
Thanks @jsotuyod !

**Comment by jsotuyod** [comment]
@KengoTODA thank you for the amazing job!

**Pull request merged**